app/controllers/api/users.php

```diff
@@ -380,6 +380,59 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             ->dynamic($user, Response::MODEL_USER);
     });
 
+App::post('/v1/users/:userId/targets')
+    ->desc('Create User Target')
+    ->groups(['api', 'users'])
+    ->label('audits.event', 'users.targets.create')
+    ->label('audits.resource', 'target/response.$id')
+    ->label('scope', 'targets.write')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'users')
+    ->label('sdk.method', 'createTarget')
+    ->label('sdk.description', '/docs/references/users/create-target.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_TARGET)
+    ->param('userId', '', new UID(), 'User ID.')
+    ->param('targetId', '', new UID(), 'Target ID.')
+    ->param('providerId', '', new UID(), 'Provider ID.')
+    ->param('identifier', '', new Text(Database::LENGTH_KEY), 'The target identifier (token, email, phone etc.)')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->inject('events')
+    ->action(function (string $userId, string $targetId, string $providerId, string $identifier, Response $response, Database $dbForProject, Event $events) {
+        $provider = $dbForProject->getDocument('providers', $providerId);
+
+        if ($provider->isEmpty()) {
+            throw new Exception(Exception::PROVIDER_NOT_FOUND);
+        }
+
+        $user = $dbForProject->getDocument('users', $userId);
+
+        if ($user->isEmpty()) {
+            throw new Exception(Exception::USER_NOT_FOUND);
+        }
+
+        $target = $dbForProject->getDocument('targets', $targetId);
+
+        if (!$target->isEmpty()) {
+            throw new Exception(Exception::USER_TARGET_ALREADY_EXISTS);
+        }
+
+        $target = $dbForProject->createDocument('targets', new Document([
+            '$id' => $targetId,
+            'providerId' => $providerId,
+            'providerInternalId' => $provider->getInternalId(),
+            'userId' => $userId,
+            'userInternalId' => $user->getInternalId(),
+            'identifier' => $identifier,
+        ]));
+        $dbForProject->deleteCachedDocument('users', $user->getId());
+        $response
+        ->setStatusCode(Response::STATUS_CODE_CREATED)
+        ->dynamic($target, Response::MODEL_TARGET);
```

Comment:
Let's add a newline and indent

```suggestion
        
        $response
            ->setStatusCode(Response::STATUS_CODE_CREATED)
            ->dynamic($target, Response::MODEL_TARGET);
```

Reply:
done this